plink-frontend/src/app/app.component.ts

```diff
@@ -1,10 +1,28 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { CurrencyService } from 'src/app/services/currency.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'Plink Frontend';
+  pricess: any;
+
+  constructor(private currencyService: CurrencyService) {
+
+  }
+
+  ngOnInit() {
+    this.getPrices();
+  }
+
+  getPrices() {
+    this.currencyService.getPrice().subscribe((data: any) => {
+      console.log(data)
+      this.pricess = data.prices;
```

Comment:
you can use destructuring. Check this [article](https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f)

Reply:
ready 💯